tests/test_timeseries.py

```diff
@@ -118,3 +115,21 @@ def test_geologic_time_series(tmp_path):
         assert ts.number_of_timestamps() == 12
         assert ((ts.timestamps[0] == -145000000 and ts.timestamps[-1] == -72100000) or
                 (ts.timestamps[0] == -145000000 * 2 and ts.timestamps[-1] == -72100000 * 2))
+
+
+def test_geologic_time_str_fails_when_not_int():
+    with pytest.raises(AssertionError):
+        rqts.geologic_time_str('hello world')
+
+
+@pytest.mark.parametrize('years_value, expected_result', [(10_000_000, '-10 Ma'), (1_000_000, '-1 Ma'),
+                                                          (11_000_000, '-11 Ma'), (1, '-0.000 Ma'),
+                                                          (1_500_000, '-1.500 Ma'), (15_060_000, '-16 Ma'),
+                                                          (15_040_000, '-16 Ma')])
```

Comment:
Interesting...we might want to change the geologic_time_str() function to round() to the nearest Ma, rather than round down.  But we can park that for a separate change some other time.